PROTOCOL.md

```diff
@@ -414,6 +417,7 @@ The following is an example `add` action:
     "dataChange": true,
     "baseRowId": 4071,
     "defaultRowCommitVersion": 41,
+    "clusteringProvider": "liquid",
```

Comment:
As https://github.com/delta-io/delta/pull/2264/files#r1393446762 says, we cannot have both `clusteringProvider` and `partitionValues` set at the same time.

Reply:
Sorry, where is that? I can't find it anywhere in this file?
https://github.com/dabao521/delta/blob/fe34ad1764a0ac212ab8c22e3d469e7f6541f598/PROTOCOL.md
(searching for all instances of `partitionValues`)

Reply:
Search for "The following is an example add action"

Reply:
I added a new writer rule that disallows partitioned table here. Let's discuss in the follow up [PR](https://github.com/delta-io/delta/pull/2294/files#diff-62d56fa1c25dda8d49ccdbf9ea31df4576f84625706d54613dff0acc8f974d97R1083). thanks